Source/Evolve/WarpXEvolve.cpp

```diff
@@ -39,8 +39,6 @@ WarpX::Evolve (int numsteps)
 
     bool early_params_checked = false; // check typos in inputs after step 1
 
-    Real walltime, walltime_start = amrex::second();
```

Comment:
Maybe you can do `static Real walltime = 0.;` instead of storing it in the WarpX class?

Reply:
That's possible, but implies that we can not create two WarpX instances, because they would share that state: https://godbolt.org/z/n675aY1ed

(There are more locations that prevent that though, I wonder if we might need to check if we want to be able to do this in the future.)

Reply:
Changed to static in a second commit for now